cluster/gce/configure-vm.sh

```diff
@@ -559,11 +559,8 @@ if [[ -z "${is_push}" ]]; then
   set-kube-env
   [[ "${KUBERNETES_MASTER}" == "true" ]] && mount-master-pd
   create-salt-pillar
-  if [[ "${KUBERNETES_MASTER}" == "true" ]]; then
-    create-salt-master-auth
```

Comment:
The error I've seen was "certificate is valid for 104.155.29.206, not 10.240.81.1". AFAIU it means the apiserver was using a certificate created for 104* IP, wheres kubelet was using 10* IP to connect to it. AFAIU it's not enough to call ```create-salt-node-auth```, but you also have to set apiserver address by calling ```salt-set-apiserver```. Am I missing something?